locales/base/translation.json

```diff
@@ -1238,6 +1238,13 @@
   "others": "Others",
   "nomSpaceRecipient": "Nomspace: {{name}}",
   "lastDay": "Last 24 hrs",
+  "merchant": "Merchant Payment",
+  "merchantMoneyEscrow": "You're about to send a direct payment to a merchant.",
+  "merchantPaymentNumberVerificationMessage": "You need to connect your number",
+  "merchantPaymentSubmitFail": "Your payment failed to submit",
+  "merchantPaymentSetup": "There was an error loading the merchant information, please try scanning the QRCode again",
+  "merchantPaymentSubmitSuccess": "Your payment successfully submitted",
```

Comment:
```suggestion
  "merchantPaymentSubmitSuccess": "Your payment was successfully submitted",
```

Reply:
Fixed in 47521465ab4da4fe5b17f591fb05d3db2b1cdc75!